incubator-tools/cmek_docai_processor/cmek_docai_processor.ipynb

```diff
@@ -0,0 +1,909 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #44.        function_build = functions_v2.BuildConfig()
</pre><p>All of these classes should be initialized with the values instead of being added in subsequent lines. It's easier to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/749/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
Updated